src/khoj/interface/web/content_type_input.html

```diff
@@ -0,0 +1,151 @@
+{% extends "base_data_integration.html" %}
+{% block content %}
+<h2>{{ content_type }}</h2>
+<form id="config-form">
+    <table>
+        <tr>
+            <th>Field</th>
+            <th>Value</th>
+        </tr>
+        <tr>
+            <td>
+                <label for="input-files">Input Files</label>
+            </td>
+            <td id="input-files-cell">
+                {% if current_config['input_files'] is none %}
+                    <input type="text" id="input-files" name="input-files">
+                {% else %}
+                    {% for input_file in current_config['input_files'] %}
+                        <input type="text" id="input-files" name="input-files" value="{{ input_file }}">
+                    {% endfor %}
+                {% endif %}
+            </td>
+            <td>
+                <button type="button" id="input-files-button">Add</button>
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <label for="input-filter">Input Filter</label>
+            </td>
+            <td id="input-filter-cell">
+                {% if current_config['input_filter'] is none %}
+                    <input type="text" id="input-filter" name="input-filter">
+                {% else %}
+                    {% for input_filter in current_config['input_filter'] %}
+                        <input type="text" id="input-filter" name="input-filter" value="{{ input_filter }}">
+                    {% endfor %}
+                {% endif %}
+            </td>
+            <td>
+                <button type="button" id="input-filter-button">Add</button>
+            </td>
+        </tr>
+    </table>
+
+    <h4>You probably don't need to edit these.</h4>
+
+    <table>
+        <tr>
+            <th>Field</th>
+            <th>Value</th>
+        </tr>
```

Comment:
Can we remove this row. Seems obvious from the UI what each of the two columns contain